manual/cops_style.md

```diff
@@ -2239,7 +2239,37 @@ some_str = 'ala' \
            'bala'
 ```
 
-## Style/MethodCallParentheses
+## Style/MethodCallWithArgsParentheses
+
+Enabled by default | Supports autocorrection
+--- | ---
+Disabled | Yes
+
+This cop checks presence of parentheses in method calls containing parameters.
+As in popular Ruby's frameworks a lot of methods should always be called without parentheses,
+users can ignore them by passing their names to IgnoredMethods option.
+
+### Example
```

Comment:
Did you edit this file manually? I'm surprised to see examples here when there are none in the cop's description.